packages/swamp/src/connectors/jsonplaceholder.ts

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
```

Comment:
What is jsonplaceholder?

Reply:
Just the name of this demo API; can kill it https://jsonplaceholder.typicode.com/